src/veins/modules/phy/Decider80211p.cc

```diff
@@ -266,7 +266,7 @@ bool Decider80211p::cca(simtime_t_cref time, AirFrame* exclude)
     bool isChannelIdle = minPower < ccaThreshold;
     if (airFrames.size() > 0) {
         size_t usedFreqIndex = airFrames.front()->getSignal().getSpectrum().indexOf(centerFrequency - 5e6);
```

Comment:
`indexOf(centerFrequency)-1` is more in line with what is used while sending (and feels more safe to me than arithmetic with `double` values)